terraform/services/rds/main.tf

```diff
@@ -0,0 +1,14 @@
+provider "aws" {
+  region = "us-east-1"
+}
```

Comment:
The aws provider block should go in a terraform.tf file alongside the backend key. See https://github.com/CMSgov/ab2d-bcda-dpc-platform/blob/main/terraform/services/github-actions-role/terraform.tf for an example.